vnext/Microsoft.ReactNative/XamlView.h

```diff
@@ -9,16 +9,13 @@ namespace Microsoft::ReactNative {
 
 using XamlView = xaml::DependencyObject;
 
-inline winrt::IPropertyValue GetTagAsPropertyValue(xaml::FrameworkElement fe) {
-  assert(fe);
-  return fe.GetValue(xaml::FrameworkElement::TagProperty()).try_as<winrt::IPropertyValue>();
-}
-
 inline int64_t GetTag(XamlView view) {
```

Comment:
nit, another option is to return optional<int64_t> which might be nicer than using -1

Reply:
That's a nice idea. Would I then return early from a function calling this and getting an empty optional (or a -1 for that matter)?

Reply:
yes same idea, instead of checking for special -1 value you just check `foo.has_value()`